build.sbt

```diff
@@ -0,0 +1,11 @@
+Global / onChangedBuildSource := ReloadOnSourceChanges
+scalaVersion := "2.13.8"
+// enablePlugins(RiddlSbtPlugin)
+libraryDependencies ++= Seq(
+  "com.reactific" %% "riddl-testkit" % "0.13.+" % "test",
```

Comment:
Biggest take away here is I need to remember to update this when we update RIDDLC...